core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/runtimeModels.scala

```diff
@@ -89,31 +87,46 @@ object RuntimeStatus extends Enum[RuntimeStatus] {
   case object Starting extends RuntimeStatus // leo
   case object Deleted extends RuntimeStatus // leo
 
-  case object Provisioning extends RuntimeStatus // gce
-  case object Staging extends RuntimeStatus // gce
-  case object Suspending extends RuntimeStatus // gce
-  case object Suspended extends RuntimeStatus // gce
-  case object Terminated extends RuntimeStatus // gce
+  def fromDataprocClusterStatus(dataprocClusterStatus: DataprocClusterStatus): RuntimeStatus =
+    dataprocClusterStatus match {
+      case DataprocClusterStatus.Creating => Creating
+      case DataprocClusterStatus.Deleting => Deleting
+      case DataprocClusterStatus.Error    => Error
+      case DataprocClusterStatus.Running  => Running
```

Comment:
Is ^^^ mapping always correct? See comment above for details.

Reply:
do you see a mapping that looks suspicious?

Reply:
I think this is fine - `ClusterMonitorActor` has knowledge about how to transition a cluster to `RuntimeStatus.Stopped` even if the `DataprocStatus` is `Running`. (Want to make sure unit and automation tests pass regarding monitoring.)